packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/lib/feature_login/blocs/login_bloc.dart

```diff
@@ -21,7 +23,9 @@ abstract class LoginBlocEvents {
   @RxBlocEvent(type: RxBlocEventType.behaviour, seed: '')
   void setPassword(String password);
 
-  void login();
+  void login();{{#enable_feature_onboarding}}
+
+  void goToRegistration();{{/enable_feature_onboarding}}
```

Comment:
Let's have the registration button in any case. If the `enable_feature_onboarding` is not enabled, let's show a bottom sheet dialog that this functionality is still not implemented

Reply:
Done.